crates/optimism/exex/src/lib.rs

```diff
@@ -116,6 +116,7 @@ where
             match &notification {
                 ExExNotification::ChainCommitted { new } => {
                     debug!(
+                        target: "trie",
```

Comment:
my bad the target should be `exex`. let's make it `optimism::exex` as well to avoid confusion